lambda-hot-reloading/javascript-terraform/lambda_src/index.js

```diff
@@ -0,0 +1,16 @@
+exports.handler = (event, context, callback) => {
+    var number1 = 21;
+    var number2 = 31;
+    var sum = number1 + number2;
+    var product = number1 * number2;
+    var difference = Math.abs(number1 - number2);
+    var quotient = number1 / number2;
+    callback(null, {
+        "Number1": number1,
+        "Number2": number2,
+        "Sum": sum,
+        "Product": product,
+        "Difference": difference,
+        "Quotient": quotient
+    });
+ };
```

Comment:
nit: could write this via an async handler to make it a bit more modern

Reply:
Will switch, it's copied from the current javascript example so I left it for now, but you are right!